src/handlers/fileResolver.js

```diff
@@ -18,6 +39,10 @@ const resolveSync = enhancedResolve.create.sync({
  */
 function resolve(relPath = "", currentFile = "") {
   const currentDir = path.dirname(currentFile);
+  if (!relPath.match(/^(\.|\/)/)) {
```

Comment:
The responsibility of the method `checkNodeModules` is exactly the same as this condition, however `checkNodeModules` also checks for windows path too (for example, `c:/`), 
I would suggest, instead of this condition use `checkNodeModules`. However the name `checkNodeModules` is not meaningful anymore, maybe rename it `isAbsolutePath`

Reply:
Just to be clear, are there any other options than these?

```
1) import foo from 'bar'
2) import foo from '../../../bar'
3) import foo from '/usr/local/npm/bar' # or 'c:/Users/foobar/npm/bar'
```

1) refers to a node module OR a babel-resolved model
2) is a relative path
3) is an absolute path

Reply:
Yeah, I guess that it covers it all:)
Just one point the first case can also be webpack alias and package.json aliases too:)

Reply:
I found another case which is `import foo from 'includes/bar` which can be resolved by babel-module-resolver, so I had to change a bit this piece of code

Reply:
I found another case which is `import foo from 'includes/bar` which can be resolved by babel-module-resolver, so I had to change a bit this piece of code

Reply:
I found another case which is `import foo from 'includes/bar` which can be resolved by babel-module-resolver, so I had to change a bit this piece of code